### ISSUE TITLE
Suggestions for speech history feature in NVDA alpha version

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#20107]
### Is your feature request related to a problem? Please describe.

Hello NVAccess team, my one suggestion for speech history functionality in NVDA alpha version. There should also be gestures to listen the previous item of speech. This helps more to listen the previous items by NVDA not to skip any part.  That's why, the gestures to move to next and previous occurance of the speech must be in NVDA. Likewise, another feature should be there to list out recent speech in a browsable dialog not just the last item. Best regards.

### Describe the solution you'd like

Explained above.

### Additional context

nvda_snapshot_alpha-55795,3ad497ae
